homeassistant/components/metoffice/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+    "config": {
+      "step": {
+        "user": {
+          "description": "The latitude and longitude will be used to find the closest weather station.",
+          "title": "Connect to the UK Met Office",
+          "data": {
+            "api_key": "Met Office DataPoint API key",
+            "latitude": "Latitude",
+            "longitude": "Longitude",
+            "name": "Friendly name"
+          }
+        }
+      },
+      "error": {
+        "cannot_connect": "Failed to connect, please try again",
+        "unknown": "Unexpected error"
+      },
+      "abort": { "already_configured": "Device is already configured" }
```

Comment:
Please use as many common keys as possible. Here are the available common keys:
https://github.com/home-assistant/core/blob/5f8dcd45c1a4279f792f39ef6deffb8f8677cef5/homeassistant/strings.json#L20-L55

Here's an example:
https://github.com/home-assistant/core/blob/5f8dcd45c1a4279f792f39ef6deffb8f8677cef5/homeassistant/components/adguard/strings.json#L7

Reply:
Done.

Reply:
Think that's there, now?